plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestingMemSqlServer.java

```diff
@@ -31,6 +31,7 @@
     private static final String MEM_SQL_LICENSE = requireNonNull(System.getProperty("memsql.license"), "memsql.license is not set");
 
     public static final String DEFAULT_TAG = "memsql/cluster-in-a-box:centos-7.1.13-11ddea2a3a-3.0.0-1.9.0";
+    public static final String LATEST_TESTED_TAG = "memsql/cluster-in-a-box:centos-7.3.4-d596a2867a-3.2.4-1.10.1";
```

Comment:
would it make sense to add a test which checks if latest did not get stale?

Reply:
This is current pattern and see https://github.com/trinodb/trino/issues/5804 for more info.

We don't automatically support newer MemSQL version when one is released, so a test verifying our Latest is truly latest could start failing any time, and it could be hard to make it fixed, if there were any breaking changes.

We should read "Latest" as  "Latest Tested" where "Tested" is omitted for brevity.